pkg/metadata.go

```diff
@@ -24,6 +24,7 @@ type NlxMetadata struct {
 	Pages              []string `json:"pages"`
 	PermissionSets     []string `json:"permissionsets"`
 	SitePermissionSets []string `json:"sitepermissionsets"`
+	Profiles           []string `json:"profiles"`
```

Comment:
Do we need this? If I'm not mistaken, we won't be supporting `profile` folders with this version of CLI (for retrieve or deploy)

Reply:
Not necessarily, but if anyone uses this CLI against an older version of pliny for some reason, having the field in will keep the JSON marshal/unmarshal from blowing up. It won't hurt to leave it in for now.

Reply:
@gzapatas I think they could just rename the folder from `profiles` to `sitepermissionsets` since its not like the internal files changed format or anything. And @acofer okay, I'm fine with that, we can just make a note to take it out later